### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by LPardue**
- Update authors
- Initial commit

### CODE REVIEW

File: draft-edm-protocol-greasing.md [hunk 1]
```diff
@@ -32,28 +31,180 @@ author:
...
+greased values by a sender for a real extension may cause a failure.
+
+Receiver implementations may unintentionally build a reliance on the occurrence of
+greasing in the temporal or spatial domain. Senders are advised to
+implementation non-determinism of when they use grease in addition to what
+values they send.
+
+ and might not be suitable for all protocols but
+where it is already defined or can be added, it is important for endpoints to
+exercise it correctly in order to achieve the maintenance goals.
```
Comment (tfpauly) [inline draft-edm-protocol-greasing.md:161]: Partial sentence?

Comment (LPardue) [reply draft-edm-protocol-greasing.md:161]: yes that should be deletec oops

File: draft-edm-protocol-greasing.md [hunk 2]
```diff
@@ -1,6 +1,5 @@
 ---
-title: "TODO - Your title"
-abbrev: "TODO - Abbreviation"
+title: "Considerations for Applying Protocol Maintenance Concepts"
```
Comment (tfpauly) [inline draft-edm-protocol-greasing.md:2]:
```suggestion
title: "Maintaining Protocols Using Grease and Variability"
```

**Review by tfpauly — APPROVED** [review]
Thank you for this initial pass! I think publishing it as something for the program to start discussing is good. I have comments on the title and abstract for how to make the scope clear and not tooo broad.

**Pull request merged**